documents-d-archeologie-francaise.csl

```diff
@@ -45,23 +48,20 @@
     <choose>
       <if variable="author">
         <names variable="author">
-          <name form="short" delimiter=", " initialize-with=". " delimiter-precedes-last="never" name-as-sort-order="all" sort-separator=" " et-al-min="3" et-al-use-first="1" font-weight="bold">
-      </name>
+          <name form="short" delimiter=", " initialize-with=". " delimiter-precedes-last="never" name-as-sort-order="all" sort-separator=" " et-al-min="3" et-al-use-first="1" font-weight="bold"/>
```

Comment:
It's generally easier to put some of these options on `<citation>` and `<bibliography>` (such as `et-al-min="3"` and `et-al-use-first="1"`) and remove them here. For a full list, see http://docs.citationstyles.org/en/stable/specification.html#inheritable-name-options.